internal/operands/data-sources/reconcile.go

```diff
@@ -39,14 +39,14 @@ func init() {
 	utilruntime.Must(cdiv1beta1.AddToScheme(common.Scheme))
 }
 
-func WatchClusterTypes() []client.Object {
-	return []client.Object{
-		&rbac.ClusterRole{},
-		&rbac.Role{},
-		&rbac.RoleBinding{},
-		&core.Namespace{},
-		&cdiv1beta1.DataSource{},
-		&cdiv1beta1.DataImportCron{},
+func WatchClusterTypes() []operands.WatchType {
+	return []operands.WatchType{
+		{Object: &rbac.ClusterRole{}, WatchFullObject: true},
+		{Object: &rbac.Role{}, WatchFullObject: true},
+		{Object: &rbac.RoleBinding{}, WatchFullObject: true},
+		{Object: &core.Namespace{}},
+		{Object: &cdiv1beta1.DataSource{}},
+		{Object: &cdiv1beta1.DataImportCron{}},
```

Comment:
`WatchFullObject = true` is the default?

Reply:
when bool is not defined, it has always value false